SU2_SOL/src/meson.build

```diff
@@ -21,6 +21,8 @@ if get_option('enable-normal')
                                         'output/filewriter/CSU2FileWriter.cpp',
                                         'output/filewriter/CSU2BinaryFileWriter.cpp',
                                         'output/filewriter/CSU2MeshFileWriter.cpp',
+                                        'output/filewriter/CParaviewXMLFileWriter.cpp',
+                                        'output/filewriter/CParaviewVTMFileWriter.cpp',
                                         'variables/CBaselineVariable.cpp',
```

Comment:
I might have missed it, but can you add these to the old build system too?

Reply:
I think this is not addressed yet.